changelogs/fragments/3875-icinga2-inv-fix.yml

```diff
@@ -0,0 +1,10 @@
+---
+breaking_changes:
```

Comment:
Breaking changes are not accepted. Add an option which allows to change the behavior, but the default **must** be the old behavior.